packages/sdk/src/evm/constants/urls.ts

```diff
@@ -61,9 +61,7 @@ export function getChainProvider(
 
   // if we still don't have an url fall back to just using the chainId or slug in the rpc and try that
   if (!rpcUrl) {
-    rpcUrl = `https://${chainId || network}.rpc.thirdweb.com/${
-      options.thirdwebApiKey || DEFAULT_API_KEY
-    }`;
+    rpcUrl = `https://${chainId || network}.rpc.thirdweb.com/${options.apiKey}`;
```

Comment:
Shouldn't we fallback `apiKey` to `thirdwebApiKey` here? - `${options.apiKey || options.thirdwebApiKey}`

Reply:
good catch!